blockchain/network.py

```diff
@@ -75,21 +74,19 @@
 
 INBOUND_TIMEOUT = 30  # seconds
 
-RECORD = 'record'
-VERIFICATION_RECORD = 'verification_record'
-VERIFICATION_INFO = 'verification_info'
-
 PHASE_1_NODE = 0b00001
 PHASE_2_NODE = 0b00010
 PHASE_3_NODE = 0b00100
 PHASE_4_NODE = 0b01000
 PHASE_5_NODE = 0b10000
 
-DATABASE_NAME = os.getenv(pg.ENV_DATABASE_NAME, pg.DEFAULT_DB_NAME)
+DATABASE_NAME = os.environ.get('BLOCKCHAIN_DB_NAME')
```

Comment:
That should not have been taken out. It should be 
DATABASE_NAME = os.getenv(pg.ENV_DATABASE_NAME, pg.DEFAULT_DB_NAME)